server/methods.js

```diff
@@ -267,8 +272,22 @@ Meteor.methods({
   },
   addMods(mods, instanceid) {
```

Comment:
@singhsanket143 can you check here if the user is allowed to add moderators?

Reply:
@peter-hank I have added the authorization check here. Please review it.

Reply:
@peter-hank Can you please review the changes and suggest if further update is required

Reply:
@peter-hank Can you please review the changes and suggest if further update is required